src/sentry/static/sentry/app/components/events/rrwebReplayer/rrWebReplayer.tsx

```diff
@@ -16,32 +33,68 @@ class RRWebReplayer extends React.Component<Props> {
 
   newRRWebPlayer: any;
 
-  async rrwebPlayer() {
+  rrwebPlayer = async () => {
+    this.setState({isLoading: true, hasError: false});
+
     const element = this.wrapperRef?.current;
 
     if (!element) {
       return;
     }
 
-    const {url} = this.props;
+    const {url, api} = this.props;
 
     try {
-      const resp = await fetch(url);
```

Comment:
`fetch` seems to be an intentional choice here, are you sure we should use our api client?